krun/platform.py

```diff
@@ -1057,26 +1058,26 @@ def _check_rmsr_loaded(self):
         stdout, stderr, rc = run_shell_cmd(cmd, failure_fatal=True)
         debug("rmsr loaded successfully.")
 
-    def _check_aslr_disabled(self):
-        debug("Checking ASLR is off")
-        with open(self.ASLR_FILE, "r") as fh:
-                enabled = fh.read().strip()
-        if enabled == "0":
-            return  # OK!
-        else:
-            # ASLR is off, but we can try to enable it
-            debug("Turning ASLR off")
-            cmd = "%s sh -c 'echo 0 > %s'" % \
-                (self.change_user_cmd, self.ASLR_FILE)
-            stdout, stderr, rc = run_shell_cmd(cmd, failure_fatal=False)
+    def _check_aslr_enabled(self):
+        """Check ASLR is cranked to level 2"""
```

Comment:
StackOverflow. You probably want to quote this file: https://www.kernel.org/doc/Documentation/sysctl/kernel.txt (grep for `randomize_va_space`)